Makefile

```diff
@@ -1,37 +1,66 @@
+# Image URL to use all building/pushing image targets
+IMG ?= fuseml-core:latest
```

Comment:
Where should it push, to github registry?

Reply:
The way it is it will push to `docker.io/library`.

The idea is to set `IMG` as environment variable and use `docker-build` to build the image with the name provided and later `docker-push` to push it to whatever registry you set on `IMG`. The later would probably be more useful for CI.